pkg/storegateway/bucket.go

```diff
@@ -1063,6 +1063,11 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 		}
 	}
 
+	if err = srv.Send(storepb.NewStatsResponse(stats.postingsFetchedSizeSum)); err != nil {
```

Comment:
If you want to track the bytes loaded from the index, then you also need to add `stats.seriesFetchedSizeSum`.

Reply:
I had to check this, but I think I follow what you're saying here, so I'll make that change!